client/src/app/site/users/models/view-user.ts

```diff
@@ -111,16 +110,6 @@ export class ViewUser extends BaseProjectableViewModel implements Searchable {
         const title = this.title ? this.title.trim() : '';
         const firstName = this.first_name ? this.first_name.trim() : '';
         const lastName = this.last_name ? this.last_name.trim() : '';
-
-        // TODO need DS adjustment first first
-        // if (this.DS.getConfig('users_sort_by').value === 'last_name') {
-        //     if (lastName && firstName) {
-        //         shortName += `${lastName}, ${firstName}`;
-        //     } else {
-        //         shortName += lastName || firstName;
-        //     }
-        // }
-
```

Comment:
Apparently, we don't want the user full names to be rearranged just because the sorting changes. `users_sort_by` is used as default sorting in dropdowns etc. now

Reply:
Can you change the text of the config variable to be more explicit, direct about this (sortig affected, not display)

Reply:
I opened #4363 to discuss which behavior we want. For me 'users_sort_by' is clear enough (I was surprised that it used to affect the display, too)